### DISCUSSION TITLE
Couple of Vorta questions

### DISCUSSION THREAD

**Discussion by alikstimix** [borgbase/vorta#2061] [General]
New user backing up home network, liking the combination of Borg/Vorta!!

Apologies if I didnt see this in the documentation, but does Vorta run borg compact after pruning?

Secondly, not sure if this is a bug or just something I am doing, but Vorta crashes reliably when I select weekly schedule of 4 weeks.   The lengthy error messages suggests an overflow. Should I document more fully and report this??

**Comment by m3nu** [comment]
Yes, this feature was recently merged, but isn't in a release yet. You can use the master branch from Github if you want to test it.

**Comment by m3nu** [comment]
When would this crash happen? When you just input 4 weeks or after? Just tried inputting and couldn't reproduce it. Hope I don't need to wait 4 weeks.

**Reply by alikstimix** [reply]
Here is the exact scenario.
I  have the pruning set to 0 hourly, 0 daily, 2 weekly, 3 monthly, 1 annually      (may or may not be relevant)
I go the schedule tab and reduce backup from every 1 weeks; to every 2 weeks; to every 3 weeks  - all good so far.
The instant I reduce to every 4 weeks I get a fatal error with message:

![Borg](https://github.com/user-attachments/assets/8175386a-bbb1-4463-9f04-2a74032f3791)

Vorta freezes, and I have to close and restart.

**Reply by alikstimix** [reply]
Oh and I am running Linux Mint 21.3 which comes with Vorta version 0.83 and borg version 1.2

**Reply by shivansh02** [reply]
> Oh and I am running Linux Mint 21.3 which comes with Vorta version 0.83 and borg version 1.2

The crashing issue was fixed in v0.8.4. Please upgrade Vorta and try again.

**Reply by alikstimix** [reply]
Great that worked - many thanks
